### ISSUE TITLE
Drop-Down Menu and menu not tabable

### ISSUE THREAD

**Issue by hmkay** [liqd/digitalstrategie#255]
**URL:** https://digitalstrategie-stage.liqd.net/
**user:** paricipant
**expected behaviour:** I can access the menue via tabbing
**behaviour:** tabbing skips the menue, I can open the drop down and the arrows but I cannot access the topics
**important screensize:** desktop
**device & browser:** Firefox, MacOS Monterey
**Comment/Question:** 
![Bildschirmfoto 2022-02-14 um 17 47 29](https://user-images.githubusercontent.com/30862422/153908171-b33491ca-9ea8-4400-9c68-4de2fa20b990.png)

**Comment by phillimorland** [comment]
NOTE: Mac issue 
@hmkay can you also see if this is issue on Windows with browserstack/Windows computer? if we are getting such differences in linux and Mac then i'm bit worried about missing Windows

**Comment by hmkay** [comment]
Otherwise I will do it next week!

**Comment by maxliqd** [comment]
On my list for tomorrow

**Comment by maxliqd** [comment]
On Windows 11, Chrome 98, screensize 1431 * 667, you can access the topics/detail pages. The blue frame seems to be too big and ist not shown completely:

<img width="600" alt="grafik" src="https://user-images.githubusercontent.com/94360651/154682187-6cceb4df-e252-4c6a-b55d-d4567d864030.png">

**Comment by maxliqd** [comment]
@phillimorland let me know if I can test anything else.
